packages/audiofileplayer/android/src/main/java/com/google/flutter/plugins/audiofileplayer/AudiofileplayerPlugin.java

```diff
@@ -42,7 +48,10 @@
  * interaction.
  */
 public class AudiofileplayerPlugin
-    implements MethodCallHandler, AudiofileplayerService.ServiceListener {
+    implements MethodCallHandler,
```

Comment:
Please test this with a v1 example app and a v2 example app and verify that everything works as expected (even when app is stopped/resumed etc).

Reply:
Yes. It works on both v1 and v2 on my end. Not sure if we need more tests though.